sites/fast-website/src/app/components/feature-card/feature-card.styles.ts

```diff
@@ -0,0 +1,69 @@
+import { css } from "@microsoft/fast-element";
+import { display, neutralForegroundRestBehavior } from "@microsoft/fast-components";
+
+export const FeatureCardStyles = css`
+    ${display("grid")} :host {
+        contain: layout;
+        grid-template-columns: 1fr 2fr;
+        color: var(--neutral-foreground-rest);
+        font-family: var(--text-font);
+        box-sizing: border-box;
+        padding: 20px;
+        box-shadow: unset;
+        height: 140px;
+        width: 600px;
+    }
+
+    :host::before {
+        content: "";
+        display: block;
+        background-color: currentColor;
+        position: fixed;
+        height: 1px;
+        width: 93%;
+        left: 20px;
+        top: -1px;
+    }
+
+    :host(:hover) ::slotted(fast-anchor) {
+        opacity: 1;
+    }
+
+    :host(:hover)::before {
+        opacity: 0;
+    }
+
+    :host(:hover) ::slotted(h5) {
+        color: var(--accent-fill-rest);
+    }
+
+    header {
+        flex: 1;
+    }
+
+    main {
+        display: flex;
+        flex-direction: column;
+        justify-content: space-between;
+        flex: 2;
+    }
+
+    ::slotted(h4) {
+        font-size: 20px;
+        margin: 0;
+    }
+
+    ::slotted(h5) {
+        margin: 0 0 10px 0;
+    }
+
+    ::slotted(p) {
+        margin: 0;
+    }
+
+    ::slotted(fast-anchor) {
+        margin-right: 20px;
+        opacity: 0;
+        color: var(--accent-fill-rest);
```

Comment:
Do we need to reset the color on the anchor? This seems like it's possibly covering up a bug.

Reply:
Is there a better option for setting the color on the anchor?  If I set appearance="accent" I do get the right color but it is a large button style appearance instead of the accent colored hypertext link.  I could use the section-anchor class that Sam added to the site stylesheet if that is preferred.

Reply:
Accent is not an anchor appearance, which is likely why this is not working. Is the goal to target the `lightweight` version?

Reply:
Looking closer at the design in Figma it could well be neutral.  Although if I set appearance to neutral or remove appearance entirely it changes the positioning slightly and stops applying the accent color.

Reply:
What color are you trying to get?

Reply:
<img width="283" alt="Screen Shot 2020-05-14 at 12 44 47 PM" src="https://user-images.githubusercontent.com/26206206/81978705-c34c6c80-95e0-11ea-81ef-18f3b8cb1a5a.png">
The accent pink here.  Which I believe is the accent pink that Jason mentioned revising in discord.

Reply:
That's the lightweight appearance.

Reply:
OK, then I have the appearance set right.  Would using the class from the main stylesheet for the site be preferred for the color?

Reply:
The color should use the accent palette, so if you set "lightweight" that should use the correct recipe to deliver an accent colored lightweight anchor depending on the contrast ratio. I guess I'm not following the issue here. The anchor should be delivering the color. If it is not, something else is going on.

Reply:
Perhaps that is yet to be set then?  If I remove the setting the text is the same as the base text off-white color.
<img width="283" alt="Screen Shot 2020-05-14 at 1 29 54 PM" src="https://user-images.githubusercontent.com/26206206/81982825-378a0e80-95e7-11ea-804a-39235701ca92.png">

Reply:
Looks like the recipe is now in place.  Removed the component color styling.